docs/pages/management/dynamic-resources/terraform-provider.mdx

```diff
@@ -191,7 +197,12 @@ To apply the configuration:
 
 ## Next steps
 
+- If you have already created Teleport resources with another tool, such as
+  `tctl`, learn how to [import Teleport resources into
+  Terraform](./import-terraform-resources.mdx).
```

Comment:
The consequences of not doing that may not be immediately clear to the reader.

Reply:
Added this to the intro of the new section I added to the setup guide.